File: 백준/Silver/1316. 그룹 단어 체커/그룹 단어 체커.swift
func solution() -> Bool {
    let word = readLine()!
    var stack: [Character] = []
    word.forEach {
        if !stack.isEmpty && stack.last! == $0 {
            stack.removeLast()
        }
        stack.append($0)
    }
    return Set(stack).count == stack.count
}
let n = Int(readLine()!)!
var count = 0
for _ in 0..<n {
    if solution() {
        count += 1
    }
}
print(count)

File: 백준/Silver/1316. 그룹 단어 체커/그룹 단어 체커.py
import sys

input = sys.stdin.readline
count = 0
for _ in range(int(input())):
    word = input().strip()
    seen = {word[0]}
    is_group_word = True

    for i in range(1, len(word)):
        if word[i] == word[i - 1]:
            continue
        if word[i] in seen:
            is_group_word = False
            break
        seen.add(word[i])

    if is_group_word:
        count += 1

print(count)